### PR TITLE
feat: Add dataset genres

### PR DESCRIPTION
**Pull request by ddeboer**
Ref netwerk-digitaal-erfgoed/network-of-terms-demo#985

### CODE REVIEW

**Comment by ddeboer** [comment]
@rschalkrce You can add the dataset genres here.

File: packages/network-of-terms-catalog/catalog/datasets/aat-materials.jsonld [hunk 1]
```diff
@@ -14,6 +14,11 @@
       "@value": "AAT - materialen"
     }
   ],
+  "genre": [
+    {
+      "@id": "https://data.cultureelerfgoed.nl/termennetwerk/onderwerpen/Abstracte-begrippen"
```
Comment (ddeboer) [inline packages/network-of-terms-catalog/catalog/datasets/aat-materials.jsonld:19]: @rschalkrce Please note that I just made this up. Could you please replace this with the real genre(s)?

**Review by rschalkrce — APPROVED** [review]
Volgens mij staat alles zo (voorlopig) goed!

**Pull request merged**